python/api/models.py

```diff
@@ -0,0 +1,353 @@
+# coding: utf-8
```

Comment:
Yea I autogenerated it using a python tool.  Originally I was going to try to convert everything to the ORM model, but then I realized that in order to do that, the full ingest process, table creation, etc would have to be done in SQLA.

I can just delete the file to avoid confusion.